candidates/views/api.py

```diff
@@ -82,6 +84,82 @@ def get(self, request, *args, **kwargs):
         )
 
 
+class CandidatesAndElectionsForPostcodeView(APIView):
+    """
+    This re-produces a lot of UpcomingElectionsView, but the output
```

Comment:
TIL: OMG, the docstring is used in the HTML version of the API! I was searching for ages about a month ago trying to figure out how to do that, and gave up. Thanks :) 
![docstring-in-drf](https://cloud.githubusercontent.com/assets/7907/23013435/7336bc7c-f422-11e6-81d9-9cb4ad28b262.png)